trieste/objectives/__init__.py

```diff
@@ -58,6 +58,20 @@
     TRID_10_MINIMIZER,
     TRID_10_MINIMUM,
     TRID_10_SEARCH_SPACE,
+    Ackley5,
```

Comment:
can we please also import here `multi_objective` ones so they are available from `objectives` package?
also interfaces?